tests/administration/users.test.js

```diff
@@ -60,18 +70,12 @@ describe('Users', () => {
       .then(() => common.createNewUser(login.name, login.realm, login.role, login.wrongpass))
       .then(() => common.logout())
       .then(() => common.login(browser, login.name, login.wrongpass, 'local' ))
-      .then(() => browser.waitForExist('//div[contains(@class,"errormessage")]',500))
+      .then(() => browser.waitForExist('//div[contains(@class,"errormessage")]',5000))
       .then(() => common.login(browser, login.name, login.pass, 'local'))
       .then(() => common.clickSidebarTab(browser, 'Users', 'System Users'));
   });
   
-
   it('should not allow deletion of current admin user', () => {
-    return browser.waitForExist(`//td[contains(text(),"admin") and position()=1]/..//button[contains(@id,"rm")]`, 500, true)
+    return browser.waitForExist(`//td[text()="admin" and position()=1]/..//button[contains(@id,"rm")]`, 5000, true)
   });
-
-  // it.only('should log out', () => {
-  //   return common.logout()
-  // })
 });
-
```

Comment:
added change to empty password test, realm test will wait until integrated scenario tests are developed